specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/examples/Monitors_GetMetricStatus_MaximumSet_Gen.json

```diff
@@ -0,0 +1,19 @@
+{
+  "operationId": "Monitors_GetMetricStatus",
+  "title": "Monitors_GetMetricStatus_MaximumSet_Gen",
+  "parameters": {
+    "api-version": "2023-04-27",
+    "subscriptionId": "nqmcgifgaqlf",
+    "resourceGroupName": "rgDynatrace",
+    "monitorName": "fhcjxnxumkdlgpwanewtkdnyuz"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "azureResourceIds": [
+          "enfghpfw"
```

Comment:
>enfghpfw [](http://example.com/codeflow?start=11&length=8)

Can you fix these and other new examples to have values closer to real values? Example: a string ARM resource ID format here?

Reply:
provided the resource id in proper format